browser/help/topics/v3-ht-annotations.md

```diff
@@ -159,17 +180,14 @@ The gnomAD v3 Hail Table annotations are defined below:
   - **n_smaller**: Count of age values falling below lowest histogram bin edge.
   - **n_larger**: Count of age values falling above highest histogram bin edge.
-- **cadd**:
-  - **phred**: Cadd Phred-like scores ('scaled C-scores') ranging from 1 to 99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values are more deleterious.
-  - **raw_score**: Raw CADD scores are interpretable as the extent to which the annotation profile for a given variant suggests that the variant is likely to be 'observed' (negative values) vs 'simulated' (positive values). Larger values are more deleterious.
-  - **has_duplicate**:Whether the variant has more than one CADD score associated with it.
-- **revel**:
-  - **revel_score**: dbNSFP's Revel score from 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious.
-  - **has_duplicate**: Whether the variant has more than one revel_score associated with it.
-- **splice_ai**:
-  - **splice_ai**: The maximum delta score, interpreted as the probability of the variant being splice-altering.
-  - **splice_consequence**: The consequence term associated with the max delta score in 'splice_ai_max_ds'.
-  - **has_duplicate**: Whether the variant has more than one splice_ai score associated with it.
-- **primate_ai**:
-  - **primate_ai_score**: PrimateAI's deleteriousness score from 0 (less deleterious) to 1 (more deleterious).
-  - **has_duplicate**: Whether the variant has more than one primate_ai_score associated with it.
+- **in_silico_predictors**: Variant prediction annotations. Struct contains prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved, or splice-altering. We chose scores for either MANE Select or canonical transcripts if a prediction score was available for multiple transcripts.
+  - **revel_max**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://www.cell.com/ajhg/fulltext/S0002-9297(16)30370-6). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors). Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai3d_max**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants. Score ranging from 0 to 1 predicting the deleteriousness of missense variants. Higher scores indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```

Comment:
```suggestion
  - **primateai3d_max**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying the pathogenicity of missense variants. Score ranges from 0 to 1, and larger values indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```